### ISSUE TITLE
ListMarginLoansService omits secType.

### ISSUE THREAD

**Issue by geniusnut** [adshao/go-binance#114]
diff --git a/margin_service.go b/margin_service.go
index e7e71f3..0078088 100644
--- a/margin_service.go
+++ b/margin_service.go
@@ -199,6 +199,7 @@ func (s *ListMarginLoansService) Do(ctx context.Context, opts ...RequestOption)
        r := &request{
                method:   "GET",
                endpoint: "/sapi/v1/margin/loan",
+               secType:  secTypeSigned,
        }
        r.setParam("asset", s.asset)
        if s.txID != nil {

**Comment by geniusnut** [comment]
sorry for the markdown grammar error.
```
diff --git a/margin_service.go b/margin_service.go
index e7e71f3..0078088 100644
--- a/margin_service.go
+++ b/margin_service.go
@@ -199,6 +199,7 @@ func (s *ListMarginLoansService) Do(ctx context.Context, opts ...RequestOption)
        r := &request{
                method:   "GET",
                endpoint: "/sapi/v1/margin/loan",
+               secType:  secTypeSigned,
        }
        r.setParam("asset", s.asset)
        if s.txID != nil {
```

**Comment by adshao** [comment]
hi @geniusnut , really appreciate if you could send a pull request for it.

**Comment by adshao** [comment]
closed as #133 merged
